index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Styles -->
+    <link rel="stylesheet" href="./assets/styles/style.css" />
+    <title>Codeeq Microverse Personal Portfolio Project</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="my-logo">My Logo</h1>
```

Comment:
- [ ] Please use the `a` tag instead of the `h2` tag. `hx` tags are meant for the main content **headings** or **subheadings** not for the logo, and it is semantically not correct which can result in *bad user experience*. You can take an example of a person and just assume that your website is bigger and have several pages and a user just lost somewhere and that person wants to go back at home page then in that case apart from `Home` navigation this `logo` with the help of `a` tag makes that person work easier to securely come back on `home-page`.